packages/eslint-plugin/src/rules/no-empty-styled-expression/__tests__/rule.test.ts

```diff
@@ -45,6 +45,29 @@ tester.run('no-styled-tagged-template-expression', noEmptyStyledExpressionRule,
         // Considered valid (not checking) due to boolean operations within styled calls not being used with empty objects
         styled.span(true && {});
     `,
+    `   
+        import { styled } from '@compiled/react';
+
+        const HeaderTitle = styled.h1<any>\`
+        \${typography.h500()}\` 
+    `,
+    `   
+        import { styled } from '@compiled/react';
+
+        const HeaderTitle = styled.h1<any>\`
+        \${typography.h500()};
+        \${typography.h400()};\` 
+    `,
+    `
+        import { styled } from '@compiled/react';
+
+        ufologger.enable();
```

Comment:
This is too specific to Atlassian, it should be renamed. Maybe one of these, or anything else you prefer.

```
object.property(); // ast
foo.bar();         // metasyntactic
```